src/commands/report.js

```diff
@@ -0,0 +1,68 @@
+'use strict'
+
+const aws = require('aws-sdk');
+const lighthouse = require('lighthouse');
+const chromeLauncher = require('chrome-launcher');
+const { v1: uuidv1 } = require('uuid');
+const config = require('../config');
+// const { categories, deviceForms } = require('../constants');
+
+const s3 = new aws.S3({
+	accessKeyId: config('AWS_ACCESS_KEY_ID'),
+	secretAccessKey: config('AWS_SECRET_ACCESS_KEY'),
+	region: config('REGION')
+});
+
+const uploadFile = file => {
+	const uniqueId = uuidv1();
+	const fileName = `report-${uniqueId}.html`;
+	const fileType = 'text/html';
+	const storageClass = 'STANDARD';
+	const acl = 'public-read'
```

Comment:
These tab indentations are bothering me! I swear on my IDE, I'm using 2-space indentations... does this happen for everyone too?

Reply:
LOL nah sis. 😂